### ISSUE TITLE
Add support for node's packages.json

### ISSUE THREAD

**Issue by EHLOVader** [zzzzzzzzzzzzzzzzzzzzzzzzzzzzzzz/no-trailing-comma#1]
Would be nice to use this in npm also

**Comment by sun** [comment]
@EHLOVader Good idea! Here is a PR: https://github.com/zzzzzzzzzzzzzzzzzzzzzzzzzzzzzzz/no-trailing-comma/pull/2. Can you help to review it and confirm that this should work as advertised in the new readme?

**Comment by EHLOVader** [comment]
Oh boy, I am sorry I missed the comment here. I did just test and neither NPM or Yarn appear to sort it, but I also couldn't simply install from the package repo. It looks good though.

I added `"zzzzzzzzzzzzzzzzzzzzzzzzzzzzzzz-no-trailing-comma": "git://github.com/zzzzzzzzzzzzzzzzzzzzzzzzzzzzzzz/no-trailing-comma.git#npm-package-sun",` to my dependencies. It installed fine.
